Dockerfile

```diff
@@ -3,12 +3,20 @@ ARG VERSION
 
 RUN apk add --no-cache git gcc musl-dev
 
+# dependencies for neo4j
+RUN apk add --update --no-cache ca-certificates cmake make g++ libressl-dev openssl-dev git curl pkgconfig
```

Comment:
Weird that adding `libressl-dev` works and using only `openssl-dev` or `libressl-dev` doesn't...

It looks like Alpine Linux [uses OpenSSL by default since 3.9](https://lists.alpinelinux.org/~alpine/devel/%3C20181011171746.4c01f758%40ncopa-desktop.copa.dup.pw%3E)  so we probably shouldn't be using LibreSSL.

Theoretically, seabolt [builds with OpenSSL on Alpine Linux 3.9](https://github.com/neo4j-drivers/seabolt/blob/1.7/ci/linux/Dockerfile.package-alpine-3.9). Not sure what's different with Alpine Linux 3.11

Let's find the correct variables and values to set so we can use OpenSSL. See:
* https://stackoverflow.com/a/16249265
* https://github.com/neo4j-drivers/seabolt/blob/1.7/cmake/Internals.cmake#L39

Reply:
Also, it looks like the built seabolt library needs to be copied over to the final Docker image:
```shell
$ docker run --rm migrate/migrate:mvid
Error loading shared library libseabolt17.so.1: No such file or directory (needed by /migrate)
<OUTPUT TRUCATED>
$
```

Reply:
Adding `COPY --from=downloader /usr/local/lib/libseabolt* /lib/` fixes the issue. I'd this before `COPY --from=downloader /go/src/github.com/golang-migrate/migrate/build/migrate.linux-386 /migrate`